reference/features/configuration.md

```diff
@@ -32,342 +30,351 @@ Server is listening on: http://127.0.0.1:7700
 
 ## Usage
 
-All options should have a value specified. Adding a command-line flag or environment variable without a value will throw an error.
+Command-line flags take precedence over environment variables. If an option is specified both as a flag and as an environment variable, MeiliSearch will use the command-line flag and its respective value.
+
+All options must specify a value. Using a command-line flag or environment variable without specifying a value will throw an error and interrupt the launch process.
 
-Command-line flags take precedence over environment variables. If an option is specified both as a flag and an environment variable, the command-line flag and its respective value will be used.
+```bash
+./meilisearch --schedule-snapshot
+```
 
-**Example**:
-`meilisearch --schedule-snapshot` throws an error.
-`meilisearch --schedule-snapshot=true` activates snapshots scheduling properly.
+```bash
+error: The argument '--schedule-snapshot <schedule-snapshot>' requires a value but none was supplied
+```
 
 ## Options
 
-#### General
+### General
 
 - [Database path](/reference/features/configuration.md#database-path)
+- [Environment](/reference/features/configuration.md#environment)
 - [HTTP address & port binding](/reference/features/configuration.md#http-address-port-binding)
 - [Master key](/reference/features/configuration.md#master-key)
-- [Environment](/reference/features/configuration.md#environment)
 
-#### Advanced
+### Advanced
 
 - [Analytics](/reference/features/configuration.md#analytics)
-- [Payload Limit Size](/reference/features/configuration.md#payload-limit-size)
-- [Snapshots](/reference/features/configuration.md#schedule-snapshot-creation):
-  - [Schedule Snapchot Creation](/reference/features/configuration.md#schedule-snapshot-creation)
```

Comment:
RIP little `snapchot`, may you rest well

Reply:
`snapchot` will always have its spot in my ❤️'s documentation